### PR TITLE
Delete AWS resources

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: clouds/aws/utilities/delete_aws_resources.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import shlex
+
+import click
+from ocp_utilities.utils import run_command
+from simple_logger.logger import get_logger
+
+from clouds.aws.aws_utils import verify_aws_credentials
+from clouds.aws.delete_s3_velero_bucket import (
+    delete_all_objects_from_s3_folder,
+    delete_bucket,
```
Comment (myakove) [inline clouds/aws/utilities/delete_aws_resources.py:11]:
```
delete_all_objects_from_s3_folder,
delete_bucket
```

should be moved to utils.

File: clouds/aws/utilities/delete_aws_resources.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        raise click.Abort()
+
+    verify_aws_credentials()
+
+    rerun_cleanup_list = clean_aws_resources(aws_regions=_aws_regions)
+    while rerun_cleanup_list:
+        rerun_cleanup_list = clean_aws_resources(aws_regions=rerun_cleanup_list)
+
+
+def clean_aws_resources(aws_regions):
```
Comment (myakove) [inline clouds/aws/utilities/delete_aws_resources.py:171]: Please add docstring that explain what is done here.

**Review by myakove — CHANGES_REQUESTED** [review]

File: clouds/aws/utilities/delete_aws_resources.py (hunk 1 above)
Comment (rnetser) [reply clouds/aws/utilities/delete_aws_resources.py:11]: done

File: clouds/aws/utilities/delete_aws_resources.py (hunk 2 above)
Comment (rnetser) [reply clouds/aws/utilities/delete_aws_resources.py:178]: done

**Pull request merged**